tests/lru_cache_test.py

```diff
@@ -49,25 +49,25 @@ class LRUCacheTest(LRUCacheTestCase):
 
   def test_get_nonexistent_key(self):
     cache = LRUCache(self.name, max_size=-1)
-    self.assertIsNone(cache.get("cache-a"))
+    self.assertIsNone(cache.get("a"))
 
   def test_put_and_get_key(self):
     cache = LRUCache(self.name, max_size=-1)
 
-    cache.put("cache-a", b"a")
-    self.assertEqual(cache.get("cache-a"), b"a")
-    self.assertEqual(set(self.path.glob("cache-*")), {self.path / "cache-a"})
+    cache.put("a", b"a")
+    self.assertEqual(cache.get("a"), b"a")
+    self.assertEqual(set(self.path.glob(f"*{_CACHE_SUFFIX}")), {self.path / f"a{_CACHE_SUFFIX}"})
```

Comment:
How about adding a helper method to do this?

```
def cache_keys(self):
  return {p.name for p in self.path.glob(f"*{_CACHE_SUFFIX}")}
```

I think you suggested this earlier and we discussed it might not be necessary, but it does look like all the `*{_CACHE_SUFFIX}` globs add too much noise, so you were right :)

Reply:
I've added a method named `assertCacheKeysEqualTo` to test this. However, I don't know if the naming is good.